inference_rules.adoc

```diff
@@ -862,6 +879,10 @@ Q: How does query batching affect the KV-cache usage?
 
 A: The size of the KV-cache is determined by the batch size. The KV-cache size can also be cached across queries, in accordance with the rule of allowing caching of sizes and shapes. Other than batching and quantization rules (that apply to activations), alternative attention mechanisms (such as paged, multi-query, sparse, group query attention, etc.) or wholesale replacement of the reference KV-cache execution are not permitted.
```

Comment:
Here it says that PagedAttention is banned, but 2 questions above it says that PagedAttention is allowed, I'm guessing this is a typo?

Also would modifications such as FlashAttention be allowed?